SyntaxBasicsHomework/ISBNControlDigitCalculation/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace ISBNControlDigitCalculation
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the first 9 digits of the ISBN: ");
+            string firstNineISBNDigits = Console.ReadLine();
+            int controlDigit = 0;
+            int multiplicationFactor = 10;
+            int digitsSum = 0;
+
+            foreach (char symbol in firstNineISBNDigits)
+            {
+                int number = int.Parse(symbol.ToString());
+
+                digitsSum += (number * multiplicationFactor);
+                multiplicationFactor--;
+            }
+
+            int remainder = digitsSum % 11;
+
+            controlDigit = 11 - remainder;
```

Comment:
controlDigit = (11 - remainder)%11; if reminder = 0 you will have controlDigit = 11, that wrong